src/Admin/MaintainAuthUsers/containers/AuthUserCreateContainer.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import ReactRouterPropTypes from 'react-router-prop-types'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router'
+import axios from 'axios'
+import { handleAxiosError, resetError, setError, setLoaded } from '../../../redux/actions/index'
+import AuthUserCreate from '../components/AuthUserCreate'
+import Page from '../../../Components/Page'
+import { validateCreate } from './AuthUserValidation'
+import { errorType } from '../../../types'
+
+class AuthUserCreateContainer extends Component {
+  constructor(props) {
+    super(props)
+    props.resetErrorDispatch()
+    this.handleChange = this.handleChange.bind(this)
+    this.handleCreate = this.handleCreate.bind(this)
+    this.state = {}
+  }
+
+  async componentDidMount() {
+    const { dispatchLoaded } = this.props
+    dispatchLoaded(true)
+  }
+
+  handleChange(event) {
```

Comment:
Not suggesting changing now but we should probably agree on which way we want to bind these when we make a new component.  I tend to write them as arrow functions (class properties) which saves having to bind them in the constructor.

Reply:
I think I tried that, but it didn't work.  Will investigate and correct in the next PR.